### PR TITLE
[SDL-0273] Add 008_Registration_appHMIType_is_array script

### PR DESCRIPTION
**Pull request by YarikMamykin**
ATF Test Scripts to check ??

This PR is **ready** for review.

### Summary
Added script that checks whether it is possible to register App with array of HMI types [WEB_VIEW, Media].

### ATF version
develop

### CLA
- [X] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/WebEngine/WebEngineProjection/008_Registration_appHMIType_is_array.lua [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+--
+-- Description:
+-- Check that it is possible to register App with array of HMI types [WEB_VIEW, Media].
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. WebEngine device is connected to SDL
+
+-- Sequence:
+-- 1. Application register with WEB_VIEW appHMIType
```
[the remark this answers is not in the document]
Comment (LitvinenkoIra) [reply test_scripts/WebEngine/WebEngineProjection/008_Registration_appHMIType_is_array.lua:13]: @aderiabin Done 
0f124d5

File: test_scripts/WebEngine/WebEngineProjection/008_Registration_appHMIType_is_array.lua [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 6
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
+config.defaultMobileAdapterType = "WS"
+common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Add AppHMIType to preloaded policy table", common.updatePreloadedPT,
+  { appSessionId, common.cloneTable(appHMIType) })
```
[the remark this answers is not in the document]
Comment (LitvinenkoIra) [reply test_scripts/WebEngine/WebEngineProjection/008_Registration_appHMIType_is_array.lua]: @aderiabin Done 
0f124d5

File: test_scripts/WebEngine/WebEngineProjection/008_Registration_appHMIType_is_array.lua [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---------------------------------------------------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (LitvinenkoIra) [reply test_scripts/WebEngine/WebEngineProjection/008_Registration_appHMIType_is_array.lua:1]: @aderiabin Done 
0f124d5

File: test_scripts/WebEngine/WebEngineProjection/008_Registration_appHMIType_is_array.lua [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description:
+-- Check that it is possible to register App with array of HMI types [WEB_VIEW, Media].
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. WebEngine device is connected to SDL
```
[the remark this answers is not in the document]
Comment (LitvinenkoIra) [reply test_scripts/WebEngine/WebEngineProjection/008_Registration_appHMIType_is_array.lua:9]: @aderiabin Done 
0f124d5

File: test_scripts/WebEngine/WebEngineProjection/008_Registration_appHMIType_is_array.lua [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+-- 3. WEB_VIEW and Media appHMITypes are allowed by policy
```
Comment (aderiabin) [inline test_scripts/WebEngine/WebEngineProjection/008_Registration_appHMIType_is_array.lua:10]: @YarikMamykin 
```suggestion
-- 3. WEB_VIEW and Media appHMITypes are allowed by policy for application (App)
```

Comment (LitvinenkoIra) [reply test_scripts/WebEngine/WebEngineProjection/008_Registration_appHMIType_is_array.lua]: @aderiabin Done in https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/30/commits/49885c343b74f7c2c2702a4e516748cfe9165450#diff-94382dcebc000bb68932a1e17bffe69cR10

File: test_scripts/WebEngine/WebEngineProjection/008_Registration_appHMIType_is_array.lua [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+--[[ Local Variables ]]
+local appSessionId = 1
+local appHMIType = { "WEB_VIEW", "MEDIA" }
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appHMIType = appHMIType
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 7
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
+config.defaultMobileAdapterType = "WS"
+common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
```
Comment (LitvinenkoIra) [inline test_scripts/WebEngine/WebEngineProjection/008_Registration_appHMIType_is_array.lua:28]: @aderiabin @YarikMamykin Do we need these lines here?

**Pull request merged**